api/server/sliceModel.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type slicer struct {
-	root string
+	root     string
```

Comment:
This commit only adds whitespace. Does not match with the title.

Reply:
agreed. cleaned alot